### PR TITLE
feat: update / upgrade npm packages & ensure consistency across

### PR DESCRIPTION
**Pull request by cksanders**
Confirm npm packages are at the latest & consistent across packages/apps to improve CI run time, reduce lock file, & get the latest features.

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -12,12 +12,13 @@
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
-    "typecheck": "tsc"
+    "tsc": "tsc"
```
Comment (Justkant) [inline website/package.json:15]: I would revert that, no need to change it IMO, it's coming from the template of docusaurus, we also use the same name of script in LL, it's clear enough and with pnpm you can already use tsc by calling `pnpm tsc`
```suggestion
    "typecheck": "tsc"
```

File: website/package.json [hunk 2]
```diff
@@ -12,12 +12,13 @@
...
   },
   "dependencies": {
     "@algolia/client-search": "^4.14.3",
     "@docusaurus/core": "2.2.0",
     "@docusaurus/preset-classic": "2.2.0",
+    "@docusaurus/theme-classic": "^2.4.0",
```
Comment (Justkant) [inline website/package.json:21]:
```suggestion
```
from the doc: https://docusaurus.io/docs/api/themes/@docusaurus/theme-classic
> If you have installed @docusaurus/preset-classic, you don't need to install it as a dependency.

File: packages/simulator/tsconfig.json [hunk 3]
```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "declaration": true,
     "declarationMap": true,
+    "ignoreDeprecations": "5.0",
```
Comment (Justkant) [inline packages/simulator/tsconfig.json:6]: Do we know what deprecation we haven't updated for ?

**Review by Justkant — COMMENTED** [review]
Nice work, just some suggestions for the docusaurus package.json and a question about `ignoreDeprecations`

File: website/package.json (hunk 1 above)
Comment (cksanders) [reply website/package.json:15]: why do you prefer the longer name, won't it be easier for a dev to type `pnpm tsc`?

File: website/package.json (hunk 2 above)
Comment (cksanders) [reply website/package.json:21]: After updating eslint & typescript there is an error if we don't add this.

File: website/package.json (hunk 1 above)
Comment (Justkant) [reply website/package.json:15]: It's not really about preferring the longer name, but more that the template comes with this command and `typecheck` is pretty standard even in LL we use the same name, also tsc and typecheck is not really the same as tsc is also a build tool

Comment (cksanders) [reply website/package.json:15]: not the same? but the value of `tsc` is `tsc`?

**Comment by Justkant** [comment]
Superseded by #170

**Pull request closed without merging**